### DISCUSSION TITLE
Metrics: Difference between OpenTelemetry.Exporter.Prometheus.AspNetCore and OpenTelemetry.Exporter.OpenTelemetryProtocol

### DISCUSSION THREAD

**Discussion by artyom-p** [open-telemetry/opentelemetry-dotnet#4495] [Q&A]
Hi, I'm quite new to monitoring and metrics in general, trying to figure out what is the difference in metrics generated by 
OpenTelemetry.Exporter.OpenTelemetryProtocol and OpenTelemetry.Exporter.Prometheus.AspNetCore.

As far as I understood the first one can be used with the OpenTelemetry collector, the application sends metrics to the collector. With the second one, Prometheus needs to scrape exposed endpoint for metrics. 

1. What is the difference comparing these two exporters, are metrics different?
2. Can I use the OpenTelemetry collector to collect the same metrics that the Prometheus exporter exposes?
3. If i will use Prometheus exporter only, does it contain same metrics Otel exporter contains?

**Comment by cijothomas** [comment]
Both are two different exporters, OTLP is "push" model, exporting metrics using OTLP format to a given OTLP Endpoint. Prometheus is a "pull" model, and exposes metrics in prometheus-native-format, and require a prometheus to 'scrape' the metrics from the app.

OTLP can operate in delta/cumulative mode, but Prometheus is cumulative only.
